internal/bminventory/inventory.go

```diff
@@ -2843,6 +2847,12 @@ func (b *bareMetalInventory) PostStepReply(ctx context.Context, params installer
 }
 
 func (b *bareMetalInventory) handleReplyError(params installer.PostStepReplyParams, ctx context.Context, log logrus.FieldLogger, h *models.Host, exitCode int64) error {
+	if exitCode == MediaDisconnected {
```

Comment:
Could it happen from any step? nothing to filter?

Reply:
It can happen on any step.